docs/csharp/how-to/compare-strings.md

```diff
@@ -28,9 +28,13 @@ equality, but some differences, such as case differences, may be ignored.
 
 ## Default ordinal comparisons
 
-The most common operations, <xref:System.String.CompareTo%2A?displayProperty=nameWithType> and
-<xref:System.String.Equals%2A?displayProperty=nameWithType> or <xref:System.String.op_Equality%2A?displayProperty=nameWithType> use
-an ordinal comparison, a case-sensitive comparison, and use the current
+The most common operations:
+
+**System.String.CompareTo(String)**,
```

Comment:
I really like that you have changed these to a list format.

However, I'd like to keep the link syntax. Can you replace this change with the following:

```md
- <xref:System.String.CompareTo%2A?displayProperty=nameWithType>
- <xref:System.String.Equals%2A?displayProperty=nameWithType>
- <xref:System.String.op_Equality%2A?displayProperty=nameWithType> 
```